README.md

```diff
@@ -1,46 +1,47 @@
 # aws-terraform-vpc\_basenetwork
 
-This module sets up basic network components for an account in a specific region. Optionally it will setup a basic VPN gateway and VPC flow logs.
+\*This module sets up basic network components for an account in a specific region. Optionally it will setup a basic VPN gateway and VPC flow logs.
```

Comment:
Looks like TFDocs had some rendering issues that need to be nudged into compliance